### ISSUE TITLE
Admin - Install Module - "Back to Module Loader" shows page with header only

### ISSUE THREAD

**Issue by kengelbr** [salesagility/SuiteCRM#1872]
#### Issue

After uploading and confirming+installing of e.g. a language module the success message appears. Below is the button "Back to Module Loader" which should lead back to the Modules Loader Overview. Instead, it leads to a blank page, only showing the header "Module Loader".
#### Expected Behavior

Button "Back to Module Loader" should lead back to the Overwiew of the Module Loader.
#### Actual Behavior

The Button "Back to Module Loader" leads to a page which shows the suiteCRM Frame around but not the full ModuleLoader page. Only the header is visible: "Module Loader". No error is shown.
#### Possible Fix

Element inspector of firefox-browser shows the following:
`form action="index.php?module=Administration&amp;view=module&amp;action=UpgradeWizard" method="post"`

`input value="Back to Module Loader" type="submit"`

`div Language Pack Installed Successfully br br input value="Back to Module Loader" type="submit" br /div br /form`

Compare to an Uninstall of a module, which works great and the button works as well, the same elements are shown in the HTML code in the browser.
#### Steps to Reproduce

1. upload e.g. a language pack
2. click install at the language pack
3. confirm the installation
4. success message is shown with button below
5. klick the "Back to Module Loader" button
#### Your Environment

- SuiteCRM Version used: 7.6.5 / 7.6.6
- Browser name and version (e.g. Chrome Version 51.0.2704.63 (64-bit)): all browsers
- Environment name and version (e.g. MySQL, PHP 7): Apapche 2 on Debian, mysql-5.5, PHP Version 5.6.17-0+deb8u1
- Operating System and version (e.g Ubuntu 16.04): Client: Windows 7 Enterprise newest updates installed, Server: Debian

> Module Loader:
> ![1_modulloader](https://cloud.githubusercontent.com/assets/4960195/17399704/e3a5d228-5a43-11e6-952a-9064a2979cd3.jpg)

---

> Confirm Installation:
> ![2_confirm](https://cloud.githubusercontent.com/assets/4960195/17399705/e3ab0b62-5a43-11e6-81b6-4512ca4e6ca8.jpg)

---

> Successful Installation:
> ![3_success](https://cloud.githubusercontent.com/assets/4960195/17399707/e3acc1d2-5a43-11e6-8e45-9b344da48407.jpg)

---

> Blank Page after Button Click
> ![4_blank_page](https://cloud.githubusercontent.com/assets/4960195/17399706/e3ab5d24-5a43-11e6-88c4-9632c9a502df.jpg)

**Comment by horus68** [comment]
I can also confirmed this issue when installing languages.
Note that sometimes it works fine... never find it why!

**Comment by kengelbr** [comment]
Thanks for confirming it!
For me it never works :(
In my opinion its a hidden FormField which has a wrong value or something. Usually a very stupid typo, since on other pages this button works fine.

**Comment by horus68** [comment]
This can hardly be considered as low priority bug by international users!

**Comment by kengelbr** [comment]
Since it's a non-blocking bug, it will have a low prio, that's quite normal. And there ist also a workaround available, so no need for a higher priority...I can understand and accept the classification!

**Comment by shogunpol** [comment]
@kengelbr , can you try to install language packs on recent version SuiteCRM(7.7.8), and confirm an issue still appear?

**Comment by shogunpol** [comment]
Thank you for raising this issue however due to community member review this issue will be closed, If this issue still exists and extra information is given that allows us to proceed with this, it will be reopened.

**Comment by kengelbr** [comment]
@shogunpol I have no 7.7.8 available for testing, but @horus68 tested it and confirmed, that the behavior still exists - he even wrote, that it sometimes works.
I'd say closing this issue might not be the right decision...what do you think?

**Comment by shogunpol** [comment]
@kengelbr , the testing on recent version of SuiteCRM(7.7.9 now), is essential. The reason is we do a lot improvements every release, and some of them are not stated. 
And yes - if you confirm is still exist on recent version i reopen an issue.

**Comment by horus68** [comment]
@shogunpol this is a strange issue, an on and off thing.
Confirmed now in 7.8 beta too.
Installed 3 language files (Portuguese, German, Italian).
The first file to install one goes fine, the others can't go back to module unless you made a refresh.
 (I also changed the order of the files to be installed).
Tested under Firefox and Crhome
![2017-01-04 13_42_54-microsoft edge](https://cloud.githubusercontent.com/assets/3055443/21644864/891d8a6e-d287-11e6-8f52-04e833e7c393.png)

The only error I got was in the Crhome console this (testsuite78beta is my instance install folder and site name)
```
Navigated to http://localhost/testsuite78beta/index.php?module=Administration&view=module&action=UpgradeWizard
index.php?module=Administration&view=module&action=UpgradeWizard:983 Resource interpreted as Document but transferred with MIME type image/gif: "http://localhost/testsuite78beta/index.php?entryPoint=getImage&imageName=blank.png".
```
![2017-01-04 13_43_15-program manager](https://cloud.githubusercontent.com/assets/3055443/21644870/90f0fe06-d287-11e6-8bf9-fe0b9519e846.png)

This error shows in Edge
![2017-01-04 14_18_39-testsuite78beta e mais 2 paginas - microsoft edge](https://cloud.githubusercontent.com/assets/3055443/21645147/cb70b5a2-d288-11e6-89be-ad644143b92b.png)
(HTTP500
HTTP500: ERROR ON SERVER - The server encountered an unexpected condition which prevented it from fulfilling the request.
POST - http://localhost/testsuite78beta/index.php?module=Administration&view=module&action=UpgradeWizard


Using SuiteCRM 7.8 beta 7 / Apache/2.4.18 (Win32) PHP/5.5.12 / 
(Wamp install for local text with full rights)
Tested in Firefox and Crhome and MS Edge

The thing is... can you test this in similar setup, install 3 language files from https://crowdin.com/project/suitecrmtranslations and the issue do not happen?
If it works for you then we need to close this and forget the issue!

**Comment by horus68** [comment]
@pgorod @kengelbr @likhobory
Can you confirm this test on other systems?
Please confirm if the button "return to module loader" works on the first language pack but not for the packs to be installed next.

**Comment by pgorod** [comment]
I don't have time to test this now, sorry...

I wonder if it is related to this issue #2788, since the error is occasional it suggests only some unlucky combination of events produces the problem. If anybody's going to test this I suggest disabling cron jobs during the tests.

**Comment by horus68** [comment]
no cronjobs active!

**Comment by likhobory** [comment]
Confirmed in 7.8 beta1 with GB & IT lang. packs. In both cases I have seen this message (FF50.1/Win10):

![ff](https://cloud.githubusercontent.com/assets/5983672/21648848/80825168-d2b0-11e6-9874-a004122f12b6.png)

**Comment by Mac-Rae** [comment]
This issue has been closed as version 7.8.x of SuiteCRM has reached its end of life and therefore will no longer be receiving updates. We have also been unable to replicate this issue in any newer version of SuiteCRM such as 7.10.x or 7.11.x however if you are able, please leave a comment explaining how and we'll re-open this issue.

For more information about 7.8.x end of life please visit our [blog post](https://suitecrm.com/7-8-x-end-of-life/)

**Comment by horus68** [comment]
@Mac-Rae the issue its also confirmed with latest version, PHP 7.2.14, MySQL 5.6.17
